av1-rtp-spec.md

```diff
@@ -292,9 +292,11 @@ The following packetization grouping would not be allowed, since it combines dat
 
 If a packet contains an OBU with an OBU extension header then the entire packet is considered associated with the layer identified by the temporal_id and spatial_id combination that are indicated in the extension header. If a packet does not contain any OBU with an OBU extension header, then it is considered to be associated with all operating points.
 
-A MANE or SFU performs its function based on a target operating point. A packet MUST be forwarded if it is associated with the target operating point, or with all operating points. A packet SHOULD NOT be forwarded if it is associated with an operating point different than the target operating point.
+The general function of a MANE or SFU is to selectively forward packets to receivers. To make forwarding decisions a MANE inspects the media payload, so that it needs to be able to parse the AV1 bitstream and cannot function when end-to-end encryption is enabled. An SFU does not parse the AV1 bitstream and therefore needs to obtain the information relevant to selective forwarding by other means, such as the Dependency Descriptor described in Appendix A. In addition to enabling bitstream-independent forwarding and support for end-to-end encryption, the Dependency Descriptor also enables forwarding where the metadata OBU provided in the AV1 bitstream is not sufficient to express the structure of the stream.

```

Comment:
...a MANE may inspect...

Reply:
If a MANE doesn't inspect the payload is it an SFU?